securedrop_client/db.py

```diff
@@ -425,6 +483,13 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return "<DraftReply {}>".format(self.uuid)
 
+    @property
```

Comment:
There are 4 **local** source collection items:
* DraftReply (not on the server)
* Reply
* Message
* File

Since a) we're displaying a source as seen if all items of that source have seen by _anyone_ at any point in history (what we're calling "global read/unread") and b) a reply has always been seen by the journalist who sent the reply, then we want to always say that a reply item has been seen.

One issue your question made me find is that I forgot to add `seen_by` to the DraftReply class, which I added for Reply, Message, and File. `seen_by` tells us whether or not a given journalist has seen a specific item, that way we know whether or not we need to tell the server to mark the item as seen by the current user. So I'll go ahead and add the fix for this.

Reply:
(note we will always say that a draft reply has been seen by the current user since we do not store seen records for draft replies)

Reply:
(in the future we may want to consider adding a local table called `seen_draft_replies` to keep the code consistent, but let's discuss this next sync and i'll add an issue for this if needed)

Reply:
I'd thought we'd decided that all Replies are always shown as `seen` by everyone? "unseen" should only be a signifier of new Source activity, imho; not newsroom activity.

Reply:
Yes, basically that was what I was saying above as to why the DraftReply and Reply objects always returning `True` when checking whether or not they have been seen.